offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -290,6 +293,16 @@ class CreatePersonView extends VerticalLayout implements Resettable{
         Validator<? extends Object> selectionValidator = Validator.from({o -> o != null}, "Please make a selection.")
 
         //Add Listeners to all Fields in the Form layout
+        this.titleField.addSelectionListener({ selection ->
+            ValidationResult result = selectionValidator.apply(selection.getValue(), new ValueContext(this.titleField))
+            if (result.isError()) {
+                createPersonViewModel.academicTitleValid = false
+                UserError error = new UserError(result.getErrorMessage())
+                titleField.setComponentError(error)
+            } else {
+                createPersonViewModel.academicTitleValid = true
+            }
+        })
```

Comment:
You have to adjust the property change listener that listens for the removal of the component errors.

Reply:
ah nvm. This was already correct.